springfox-swagger-common/src/main/java/springfox/documentation/swagger/schema/ApiModelBuilder.java

```diff
@@ -36,10 +41,13 @@
 @Order(SwaggerPluginSupport.SWAGGER_PLUGIN_ORDER)
 public class ApiModelBuilder implements ModelBuilderPlugin {
   private final TypeResolver typeResolver;
+  private final ModelProvider modelProvider;
 
   @Autowired
-  public ApiModelBuilder(TypeResolver typeResolver) {
+  public ApiModelBuilder(TypeResolver typeResolver,
+                         @Qualifier("cachedModels") ModelProvider modelProvider) {
```

Comment:
Ideally we don't want plugins to depend on other plugins. While this works this has the possibility of introducing circular dependencies